mindjunkies/forums/views.py

```
@@ -1,129 +1,176 @@
 from django.shortcuts import get_object_or_404, redirect, render
-from django.urls import reverse
+from django.views import View
 from django.views.generic import TemplateView
 
-from mindjunkies.courses.models import Course
+from mindjunkies.courses.models import Course, Module
 
-from .forms import ForumReplyForm, ForumTopicForm
-from .models import ForumReply, ForumTopic
+from .forms import ForumCommentForm, ForumReplyForm, ForumTopicForm
+from .models import Course, ForumComment, ForumTopic, Reply
```

Ruff (0.8.2)

Redefinition of unused `Course` from line 7

(F811)